### DISCUSSION TITLE
No inverters configured in FoxESS - Modbus

### DISCUSSION THREAD

**Discussion by Alllfff** [nathanmarlor/foxess_modbus#575] [Support] [answered]
My fox modbus integration works fine but the charge period states no inverter configured.

I've tried restarting HA and reload the front end.

**Comment by FozzieUK** [comment]
Couple of questions, 
in the charge card editor have you selected the inverter in the drop down.
In Settings, Devices & Services, Foxess - Modbus  Do you have any other devices listed ?, particularly ‘disabled’ ones

**Reply by Alllfff** [reply]
Yes I have selected it in the drop down menu
I don't understand the second question, I have 1 device with 89 entities, all enabled and showing values where value available...this card worked for months, I also used it in automation, only stopped working today

**Reply by canton7** [reply]
Can you screenshot the error you're seeing, please?

**Reply by Alllfff** [reply]
![Screen Shot 2024-03-20 at 20 41 43](https://github.com/nathanmarlor/foxess_modbus/assets/39865207/40cb6c4e-1151-428c-aef5-8a3ae5a517f4)
![Screen Shot 2024-03-20 at 20 41 29](https://github.com/nathanmarlor/foxess_modbus/assets/39865207/8317e9d6-e8c9-4341-8116-9269efb7a5b6)

**Reply by canton7** [reply]
What version of foxess - modbus are you using?

**Reply by Alllfff** [reply]
v1.10.2b3

**Reply by canton7** [reply]
Does it work if you downgrade to the latest stable version, v1.10.1? Or v1.9.0 perhaps?

**Reply by Alllfff** [reply]
works ok in v1.10.2b2

**Reply by canton7** [reply]
OK, looks like I've broken something thanks! I'll take a look. It'll probably be in a few days I'm afraid - got a busy few coming up.

**Reply by canton7** [reply]
See #576

**Reply by Alllfff** [reply]
no worries, thank you for the work you do!

**Comment by canton7** [comment]
What inverter do you have?

**Reply by Alllfff** [reply]
AC1-5.0-E

**Comment by canton7** [comment, marked as answer]
This should be fixed in the latest beta, sorry about that!
